Apps/Patient/src/Models/Address.cs

```diff
@@ -0,0 +1,75 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.Patient.Models;
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+/// <summary>
+/// Represents an address.
+/// </summary>
+public class Address
+{
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Address"/> class.
+        /// </summary>
+        public Address()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Address"/> class.
+        /// </summary>
+        /// <param name="address">The address model object from patient service model.</param>
+        public Address(Common.Models.Address address)
+        {
+            this.StreetLines = address.StreetLines;
+            this.City = address.City;
+            this.State = address.State;
+            this.PostalCode = address.PostalCode;
+            this.Country = address.Country;
+        }
+
+        /// <summary>
+        /// Gets the street lines.
+        /// </summary>
+        [JsonPropertyName("streetLines")]
```

Comment:
I had a chat with Brian and I think we should reuse the existing Address model in Common/Models instead of creating a new model.  The properties for that are the same as the ones here and I think they work well for the most part.

Reply:
Reusing Common.Models.Address now so will use existing string array for street lines.